app/static/css/embed_widget.css

```
@@ -1,45 +1,78 @@
   font-size: 11px;
+  letter-spacing: .02em;
   color: var(--embed-muted);
 }
-.embed-status { font-size: 11px; font-weight: 700; letter-spacing: .03em; }
+.embed-status {
+  padding: 4px 8px;
+  border-radius: 999px;
+  font-size: 11px;
+  font-weight: 700;
+  letter-spacing: .03em;
+  border: 1px solid currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)